src/applications/post-911-gib-status/containers/Post911GIBStatusApp.jsx

```diff
@@ -36,7 +38,18 @@ function AppContent({ children, isDataAvailable }) {
   return <div>{view}</div>;
 }
 
+AppContent.propTypes = {
+  children: PropTypes.node,
+  isDataAvailable: PropTypes.bool,
+};
+
 function Post911GIBStatusApp({ user, children }) {
+  const { useToggleValue, TOGGLE_NAMES } = useFeatureToggle();
```

Comment:
This is the meat and potatoes of the PR.  Check if the toggle is enabled, if it is, we pass `"v1"` as the api version.  otherwise `"v0"` (current behavior)